kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/TopicsResourceIntegrationTest.java

```diff
@@ -184,4 +185,71 @@ public void getTopic_nonExistingTopic_returnsNotFound() {
         request("/v3/clusters/" + clusterId + "/topics/foobar").accept(Versions.JSON_API).get();
     assertEquals(Status.NOT_FOUND.getStatusCode(), response.getStatus());
   }
+
+  @Test
+  public void createTopic_nonExistingTopic_returnsCreatedTopic() throws Exception {
+    String baseUrl = restConnect;
+    String clusterId = getClusterId();
+    String topicName = "topic-4";
+
+    String expected =
+        OBJECT_MAPPER.writeValueAsString(
+            new GetTopicResponse(
+                new TopicData(
+                    new ResourceLink(
+                        baseUrl + "/v3/clusters/" + clusterId + "/topics/" + topicName),
+                    clusterId,
+                    topicName,
+                    /* isInternal= */ false,
+                    /* replicationFactor= */ 1,
+                    new Relationship(
+                        baseUrl
+                            + "/v3/clusters/" + clusterId
+                            + "/topics/" + topicName
+                            + "/configurations"),
+                    new Relationship(
+                        baseUrl
+                            + "/v3/clusters/" + clusterId
+                            + "/topics/" + topicName
+                            + "/partitions"))));
+
+    Response response =
+        request("/v3/clusters/" + clusterId + "/topics")
+            .accept(Versions.JSON_API)
+            .post(
+                Entity.entity(
+                    "{\"data\":{\"attributes\":{\"topic_name\":\""
+                        + topicName + "\",\"partitions_count\":1,\"replication_factor\":1}}}",
+                    Versions.JSON_API));
+    assertEquals(Status.CREATED.getStatusCode(), response.getStatus());
+    assertEquals(expected, response.readEntity(String.class));
```

Comment:
it would be nice to actually verify the topic is created as well.

Reply:
Done.

Reply:
Even more done.